care/facility/tests/test_bed_api.py

```diff
@@ -13,26 +14,147 @@ def setUpTestData(cls) -> None:
         cls.local_body = cls.create_local_body(cls.district)
         cls.super_user = cls.create_super_user("su", cls.district)
         cls.facility = cls.create_facility(cls.super_user, cls.district, cls.local_body)
+        cls.facility2 = cls.create_facility(
+            cls.super_user, cls.district, cls.local_body
+        )
         cls.asset_location = cls.create_asset_location(cls.facility)
+        cls.asset_location2 = cls.create_asset_location(cls.facility2)
         cls.user = cls.create_user("staff", cls.district, home_facility=cls.facility)
         cls.patient = cls.create_patient(
             cls.district, cls.facility, local_body=cls.local_body
         )
-
-    def setUp(self) -> None:
-        super().setUp()
-        self.bed1 = Bed.objects.create(
-            name="bed1", location=self.asset_location, facility=self.facility
+        cls.bed1 = Bed.objects.create(
+            name="bed1", location=cls.asset_location, facility=cls.facility
         )
-        self.bed2 = Bed.objects.create(
-            name="bed2", location=self.asset_location, facility=self.facility
+        cls.bed2 = Bed.objects.create(
+            name="bed2", location=cls.asset_location, facility=cls.facility, bed_type=1
         )
-        self.bed3 = Bed.objects.create(
-            name="bed3", location=self.asset_location, facility=self.facility
+        cls.bed3 = Bed.objects.create(
+            name="bed3", location=cls.asset_location2, facility=cls.facility2
```

Comment:
moving this to setUpTestData, will create the objects only once, this could cause issues if you expect the objects to not change